### PR TITLE
SHO-78: Information page Hero

### PR DESCRIPTION
**Pull request by saulprl**
## Describe your changes

This PR adds a new route and a new component: `/frecuentes` and `<RegularsHero />`.

## Task URL (Asana, Jira, etc...)

[SHO-78](https://cultoperrocafe.nifty.pm/l/7BNceFTvXOh8Fv)

## What was done in this pull request

- [x] Added new component: `<RegularsHero />`.
- [x] Added new route: `/frecuentes` or `frecuentes.tsx`
- [x] Storybook with RegularsHero's only variant.
- [x] Added to `frecuentes.tsx`.

## Demo / Screenshot / Video

[SHO-78.webm](https://user-images.githubusercontent.com/39208827/235803500-adbd84b8-50e0-4f1c-a230-66e7cbd179dc.webm)

### CODE REVIEW

File: app/routes/frecuentes.tsx [hunk 1]
```diff
@@ -0,0 +1,25 @@
+import {Link} from '@remix-run/react';
+
+import {NavBar, NavBarLink} from '~/components/organisms/navbar';
+import {RegularsHero} from '~/components/organisms/regulars-hero/regulars-hero.component';
+import {Footer} from '~/components/templates/footer';
+import configData from '~/config.json';
+
+// @ts-ignore
```
Comment (arielplas) [inline app/routes/frecuentes.tsx:8]: Why is `@ts-ignore` needed here?

File: app/components/organisms/regulars-hero/regulars-hero.styles.tsx [hunk 2]
```diff
@@ -0,0 +1,93 @@
...
+
+export const SecondaryContainer = styled.div`
+  position: relative;
+  border-bottom: 4px solid ${select(({colors}) => colors.black)};
+
+  ${desktopMediaQuery} {
+    flex: 1;
+    border: 4px solid ${select(({colors}) => colors.black)};
+  }
+`;
```
Comment (arielplas) [inline app/components/organisms/regulars-hero/regulars-hero.styles.tsx]: in the main page i believe we use `3px` for the border's `width`. if there's a reason to use `4px` let me know!

**Review by imariel2d — COMMENTED** [review]
Please review comments!

File: app/routes/frecuentes.tsx (hunk 1 above)
Comment (saulprl) [reply app/routes/frecuentes.tsx:8]: The usage of this `@ts-ignore` comes from an early version of the `NavBar` component created by Tony. Remix Run's `Link`s are incompatible with Storybook, and so this workaround was made. I tried creating a type for it on the NavBar's final version and it just seemed too difficult for me, honestly.

File: app/components/organisms/regulars-hero/regulars-hero.styles.tsx (hunk 2 above)
Comment (saulprl) [reply app/components/organisms/regulars-hero/regulars-hero.styles.tsx]: That should actually be using the `borderWidth` prop from the theme, thanks for pointing it out!

**Review by imariel2d — APPROVED** [review]

**Pull request merged**